service/middleware/access_log.go

```diff
@@ -38,11 +39,24 @@ func NewAccessLog() (*AccessLog, error) {
 	return &AccessLog{}, nil
 }
 
+func (a *AccessLog) ignore(req *rest.Request) bool {
+	// ignore liveness and readiness probes
+	if strings.Compare(req.URL.RequestURI(), "/status") == 0 {
```

Comment:
```suggestion
	if req.URL.RequestURI() == "/status") {
```

From the [go documentation](https://golang.org/pkg/strings/#Compare):
> Compare is included only for symmetry with package bytes. It is usually clearer and always faster to use the built-in string comparison operators ==, <, >, and so on